share/picongpu/examples/Bremsstrahlung/etc/picongpu/1.cfg

```diff
@@ -52,11 +52,16 @@ TBG_ph_calorimeter="--ph_calorimeter.period 1000 --ph_calorimeter.openingYaw 360
 
 TBG_ph_energyHistogram="--ph_energyHistogram.period 1000 --ph_energyHistogram.filter all --ph_energyHistogram.minEnergy 10 --ph_energyHistogram.maxEnergy 10000"
 
-TBG_plugins="--hdf5.period 1000 --hdf5.file simData \
-             --e_macroParticlesCount.period 1000 \
-             --i_macroParticlesCount.period 1000 \
+# file I/O with openPMD-HDF5
+TBG_openPMD="--openPMD.period 1000  \
+             --openPMD.file simData \
+             --openPMD.ext h5"
```

Comment:
Note: probably ideally we would like to go to automatic backend at some time in the future, so that the .cfg files work out-of-the-box for more users. However, not in this PR.